### DISCUSSION TITLE
curl to list the models available via cloud connections?

### DISCUSSION THREAD

**Discussion by raymondclowe** [open-webui/open-webui#7798] [Q&A]
Using open webai I can get a list of all the models running locally like this:

curl http://localhost:3000/ollama/v1/models

But can I also get a list of the models avialable via cloud connections? And can I call chat completion on those via localhost?

**Comment by avviano** [comment]
Same question here. I have 400 models showing up via four APIs and around 50 of them have no prefix but are external. No idea which API they are connected to, making model management very confusing. When I click on model properties (for external ones), there is nothing about which API they are related too. Prefixes work for some but not all.
